### ISSUE TITLE
Job Failure

### ISSUE THREAD

**Issue by anthonytec2** [deepkit/deepkit#18]
I tried to run a job using deepkit and it never starts up, I get the bellow error on the node:
```
Start job task 127.0.0.1 6c49f921-fa20-476e-aac3-cbb8821923d9 main 0
Job errored 6c49f921-fa20-476e-aac3-cbb8821923d9 main 0 Error: Command failed with exit code 1 (EPERM): /home/anthony/deepkit/cli/bin/node /home/anthony/deepkit/cli/main.js start 127.0.0.1 8961 6c49f921-fa20-476e-aac3-cbb8821923d9 81cb8bd7-2bcf-4219-913c-c3ddb668b72c main 0 eyJlbnYiOltdLCJkb2NrZXJCaW5kcyI6W10sImhvc3RFeGVjdXRpb25BbGxvd2VkIjpmYWxzZSwiY3VzdG9tTW91bnRzQWxsb3dlZCI6ZmFsc2V9
    at e.exports (/home/anthony/deepkit/cli/main.js:180:176343)
    at /home/anthony/deepkit/cli/main.js:174:73052
    at processTicksAndRejections (internal/process/task_queues.js:85:5) {
  command: '/home/anthony/deepkit/cli/bin/node /home/anthony/deepkit/cli/main.js start 127.0.0.1 8961 6c49f921-fa20-476e-aac3-cbb8821923d9 81cb8bd7-2bcf-4219-913c-c3ddb668b72c main 0 eyJlbnYiOltdLCJkb2NrZXJCaW5kcyI6W10sImhvc3RFeGVjdXRpb25BbGxvd2VkIjpmYWxzZSwiY3VzdG9tTW91bnRzQWxsb3dlZCI6ZmFsc2V9',
  exitCode: 1,
  exitCodeName: 'EPERM',
  stdout: "args['start-config-base64'] eyJlbnYiOltdLCJkb2NrZXJCaW5kcyI6W10sImhvc3RFeGVjdXRpb25BbGxvd2VkIjpmYWxzZSwiY3VzdG9tTW91bnRzQWxsb3dlZCI6ZmFsc2V9\n" +
    'jobFiles /home/anthony/.deepkit/cli/job/6c49f921-fa20-476e-aac3-cbb8821923d9/instance-checkout/main/0\n' +
    'Checkout files',
  stderr: '(node:11487) [DEP0005] DeprecationWarning: Buffer() is deprecated due to security and usability issues. Please use the Buffer.alloc(), Buffer.allocUnsafe(), or Buffer.from() methods instead.\n' +
    'Error: unknown operator: $or\n' +
    '    at Object.t.getUnserializedError (~/deepkit/cli/main.js:50:70725)\n' +
    '    at t._next (~/deepkit/cli/main.js:174:277534)\n' +
    '    at t.__tryOrUnsub (~/deepkit/cli/main.js:16:61380)\n' +
    '    at t.next (~/deepkit/cli/main.js:16:60525)\n' +
    '    at t._next (~/deepkit/cli/main.js:16:59593)\n' +
    '    at t.next (~/deepkit/cli/main.js:16:59264)\n' +
    '    at c.t.next (~/deepkit/cli/main.js:16:74263)\n' +
    '    at Object.replies.<computed> (~/deepkit/cli/main.js:174:267843)\n' +
    '    at t.SocketClient.onDecodedMessage (~/deepkit/cli/main.js:174:263759)\n' +
    '    at t.Batcher.messageComplete (~/deepkit/cli/main.js:180:6014)\n' +
    '    at ORIGIN (action job.getJobFiles)\n' +
    '    at e.exports.<anonymous> (/app/main.js:2:1330616)\n' +
    '    at A (/app/main.js:2:1322204)\n' +
    '    at Socket.<anonymous> (/app/main.js:2:1325896)\n' +
    '    at Object.t.asyncOperation (~/deepkit/cli/main.js:113:127820)\n' +
    '    at t.SocketClient.stream (~/deepkit/cli/main.js:174:266228)\n' +
    '    at Proxy.<anonymous> (~/deepkit/cli/main.js:174:263459)\n' +
    '    at ~/deepkit/cli/main.js:441:344226',
  all: '(node:11487) [DEP0005] DeprecationWarning: Buffer() is deprecated due to security and usability issues. Please use the Buffer.alloc(), Buffer.allocUnsafe(), or Buffer.from() methods instead.\n' +
    "args['start-config-base64'] eyJlbnYiOltdLCJkb2NrZXJCaW5kcyI6W10sImhvc3RFeGVjdXRpb25BbGxvd2VkIjpmYWxzZSwiY3VzdG9tTW91bnRzQWxsb3dlZCI6ZmFsc2V9\n" +
    'jobFiles /home/anthony/.deepkit/cli/job/6c49f921-fa20-476e-aac3-cbb8821923d9/instance-checkout/main/0\n' +
    'Checkout files\n' +
    'Error: unknown operator: $or\n' +
    '    at Object.t.getUnserializedError (~/deepkit/cli/main.js:50:70725)\n' +
    '    at t._next (~/deepkit/cli/main.js:174:277534)\n' +
    '    at t.__tryOrUnsub (~/deepkit/cli/main.js:16:61380)\n' +
    '    at t.next (~/deepkit/cli/main.js:16:60525)\n' +
    '    at t._next (~/deepkit/cli/main.js:16:59593)\n' +
    '    at t.next (~/deepkit/cli/main.js:16:59264)\n' +
    '    at c.t.next (~/deepkit/cli/main.js:16:74263)\n' +
    '    at Object.replies.<computed> (~/deepkit/cli/main.js:174:267843)\n' +
    '    at t.SocketClient.onDecodedMessage (~/deepkit/cli/main.js:174:263759)\n' +
    '    at t.Batcher.messageComplete (~/deepkit/cli/main.js:180:6014)\n' +
    '    at ORIGIN (action job.getJobFiles)\n' +
    '    at e.exports.<anonymous> (/app/main.js:2:1330616)\n' +
    '    at A (/app/main.js:2:1322204)\n' +
    '    at Socket.<anonymous> (/app/main.js:2:1325896)\n' +
    '    at Object.t.asyncOperation (~/deepkit/cli/main.js:113:127820)\n' +
    '    at t.SocketClient.stream (~/deepkit/cli/main.js:174:266228)\n' +
    '    at Proxy.<anonymous> (~/deepkit/cli/main.js:174:263459)\n' +
    '    at ~/deepkit/cli/main.js:441:344226',
  failed: true,
  timedOut: false,
  isCanceled: false,
  killed: false,
  signal: undefined

```

**Comment by marcj** [comment]
Thanks for the report. This is a known issue and is already fixed in master. Will be part of the next release, which happens ASAP.

**Comment by marcj** [comment]
This has been fixed in version 2020.3.1.
